pimd/pim6_mld.c

```diff
@@ -1056,6 +1057,13 @@ static void gm_handle_v1_report(struct gm_if *gm_ifp,
 	gm_ifp->stats.rx_old_report++;
 
 	hdr = (struct mld_v1_pkt *)data;
+	if (pim_addr_cmp(hdr->grp, *pkt_dst)) {
+		if (PIM_DEBUG_GM_PACKETS)
+			zlog_debug(
+				log_pkt_dst("malformed MLDv1 report (wrong destination address)"));
+		gm_ifp->stats.rx_drop_malformed++;
```

Comment:
add grp, dst, and ifname to the debug message plz.

Reply:
Done. Thanks.

```
2025/07/16 23:17:08 PIM6: [X75F2-2KZYG] [MLD default:enp1s0 ff99::99] malformed MLDv1 report (destination address should be ff99::98)
```